src/router_core/agent_connection.c

```diff
@@ -240,6 +245,17 @@ static void qdr_connection_insert_column_CT(qdr_core_t *core, qdr_connection_t *
         qd_compose_insert_string(body, text);
         break;
 
+    case QDR_CONNECTION_UPTIME_SECONDS:
+        qd_compose_insert_uint(body, core->uptime_ticks - conn->conn_uptime);
+        break;
+
+    case QDR_CONNECTION_LAST_DLV_SECONDS:
+        if (conn->is_delivery_sent)
+            qd_compose_insert_uint(body, core->uptime_ticks - conn->last_delivery_time);
+        else
+            qd_compose_insert_string(body, "-");
```

Comment:
Adding hyphens to fields is the job of qdstat.  It should not be done here (I don't believe there is a precedent for it).  A better option is to insert a null value if there is no information to send.

Reply:
Changed this to insert null instead of "-"